config/initializers/geocoder.rb

```diff
@@ -0,0 +1,21 @@
+Geocoder.configure(
+  # geocoding options
+  # :timeout      => 3,           # geocoding service timeout (secs)
+   :lookup       => :google,     # name of geocoding service (symbol)
```

Comment:
Unnecessary spacing detected.<br>Indent the first parameter one step more than the start of the previous line (not counting the comment).<br>Use the new Ruby 1.9 hash syntax.